lib/tasks/setup.rake

```diff
@@ -82,6 +84,8 @@ DESC
       user.save
       puts "User #{user.username} created successfully"
 
+      user.get_auth_token
```

Comment:
Should we have it before save, that way it can be stored, but it depends on what get_auth_token does?

Reply:
get_auth_token calls save, so it can be after user.save. no issues there.

Reply:
Maybe we can add it as a `before_create` filter in `app/models/user.rb` and `app/models/carto/user.rb`

```ruby
before_create get_auth_tokens
```

That should do and should work for all cases. What do you think?